### ISSUE TITLE
Excluded from Hactoberfest 2019

### ISSUE THREAD

**Issue by frogo123** [s-bridges/Hacktoberfest_Fizzbuzz#59]
This is the message I received when completing the challenge and sending a pull request.
Your PR was submitted to a repository that we've excluded from Hacktoberfest as it doesn't align with our core values.

**Comment by PucklaJ** [comment]
The @hacktoberfest-team opened [an issue](https://github.com/s-bridges/Hacktoberfest_Fizzbuzz/issues/44) about that themselves.

**Comment by s-bridges** [comment]
Yes that is accurate. They've decided to halt any and all things that are short of you working on an actual professional open source project. They don't want to send out shirts this year lol
